robottelo/ui/factory.py

```diff
@@ -582,6 +582,19 @@ def edit_param(session, org=None, loc=None,  force_context=True, **kwargs):
     Settings(session.browser).update(**update_args)
 
 
+def remove_param(session, org=None, loc=None, force_context=True, **kwargs):
```

Comment:
still no reasons to put that method to factory module

Reply:
Actually remove it completely and use `remove_parameter` directly with adding navigation to entity method